examples/micronaut/src/main/groovy/example/micronaut/HelloController.groovy

```diff
@@ -0,0 +1,14 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+
+@CompileStatic
+@Controller("/hello") // <1>
+class HelloController {
+    @Get("/") // <2>
+    String index() {
+        "Hello Jib" // <3>
```

Comment:
Everything on the README says the output is "Hello World", maybe this should match that.

Reply:
Fixed